monix-eval/shared/src/main/scala/monix/eval/Coeval.scala

```diff
@@ -771,15 +775,15 @@ sealed abstract class Coeval[+A] extends (() => A) with Serializable { self =>
     * of 10 times before giving up:
     *
     * {{{
-    *   import scala.concurrent.duration._
+    *   val fa = Coeval(78)
```

Comment:
This could use a better example, e.g. with `Random.nextInt(20)` failures and a `var`, so it is actually visible what's going on.